transaction-client/src/main/java/com/yahoo/omid/transaction/PostCommitActions.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2011-2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.omid.transaction;
+
+import com.yahoo.omid.tsoclient.CellId;
+
+public interface PostCommitActions {
```

Comment:
This interface has 3 methods, the last one is a combination of 1 and 2.
An alternative interface would look like

```
ListenableFuture updateShadowCells(AbstractTransaction<? extends CellId> transaction) throws TransactionManagerException;
ListenableFuture removeCommitTableEntry(AbstractTransaction<? extends CellId> transaction) throws TransactionManagerException;
```
so that implementation can be sync/async but our code decides whether to wait or not